<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadTeamsBtn">
        <property name="text">
         <string>Load teams</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>displayName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>created</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="text">
         <string>Archive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unArchiveBtn">
        <property name="text">
         <string>unArchive</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="nameFilter_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="filterName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dateFilter">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>filter date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="filterTime"/>
       </item>
       <item>
        <widget class="QRadioButton" name="beforeOrAfter">
         <property name="text">
          <string>before</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="filterRun">
       <property name="text">
        <string>Run filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
